### PR TITLE
Denoise function

### PR DESCRIPTION
**Pull request by JiatengTao**
Denoise function

### CODE REVIEW

**Comment by JiatengTao** [comment]
Denoise beta1

File: nextpow2.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import numpy as np
```
Comment (aidanjgriffiths) [inline nextpow2.py:1]: Could this fit in `denoise.py` instead? I think its a bit out of place to define 1 function in a single script.

**Comment by aidanjgriffiths** [comment]
I don't think we'll need a `denoise.wav` file the repository. I think we should avoid binary files were possible.

File: nextpow2.py (hunk 1 above)
Comment (JiatengTao) [reply nextpow2.py:1]: Sorry  I will solve it and delete the wav files

**Pull request closed without merging**